vestido/vestido-storefront/modules/ProductListView/AddToCartButton.tsx

```diff
@@ -39,14 +44,43 @@ const AddToCartButton: React.FC<AddToCartButtonProps> = ({
       // setSuccess(false);
       // Reset success state when a new action starts
 
+      const images = (item.images ?? []) as ImageSchemaType[];
+
       try {
         await trigger({
           itemId: item.id,
           qty: qty,
           variantId: defaultVar?.id ?? product?.variants?.[0]?.id ?? null,
         });
-        // setSuccess(true);
-        // Show success state after adding
+
+        toast({
+          title: '',
+          description: (
+            <>
+              <div className="flex font-semibold text-xl items-center gap-2 mb-3">
+                <LuCheckCircle className="text-green-500" strokeWidth={3} />
```

Comment:
I see that this particular JSX has been duplicated everywhere in this PR. Can we please make a function that returns this JSX?